cmake/toolchainFiles/toolchain_x86_64.cmake

```diff
@@ -2,17 +2,5 @@
 # Copyright (C) 2022, Advanced Micro Devices, Inc. All rights reserved.
 # SPDX-License-Identifier: MIT
 
-# specify the compiler
-set(CLANG_VER 12)
```

Comment:
Great ! This one was a pain to me about looking for such a compiler in an antique boutique.